submissions/evgenii-del/oop-exercise/index.js

```diff
@@ -0,0 +1,96 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/evgenii-del/a-tiny-JS-world
+   Web app: http://crooked-discovery.surge.sh/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Inhabitant {
+    constructor(species, name, gender, saying) {
+        this.species = species;
+        this.name = name;
+        this.gender = gender;
+        this.saying = saying;
+    }
+
+    toString(object) {
+        return [this.species, this.name, this.gender, this.saying].join(';');
+    }
+}
+
+class Man extends Inhabitant {
+    constructor(species, name, gender, saying, hands, legs) {
+        super(species, name, gender, saying);
+        this.hands = hands;
+        this.legs = legs;
+    }
+
+    toString(object) {
+        return super.toString() + [this.hands, this.legs].join(';');
+    }
+}
+
+class Woman extends Inhabitant {
+    constructor(species, name, gender, saying, hands, legs) {
```

Comment:
`constructor` is a special function, yet a function, which means we can define default values for some of the params that are least unique.